### PR TITLE
Cleaning

### PR DESCRIPTION
**Pull request by tloubrieu-jpl**
## 🗒️ Summary
I removed all the submodule which are not necessary here.

I updated the docker-compose to enable to launch all the service with test data for integration tests. I also set the docker images without version so that it takes the latest by default.

Note that I pushed the docker images on docker hub: https://hub.docker.com/u/nasapds so that the user don't need to build them locally.

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -31,6 +31,15 @@ docker-compose --profile=services up -d
...
 ```
 
+For test, start the registry with some test data loaded:
+
+    docker-compose --profile=integration-test up
+
+
+For API dev, start the registry with some test data, without the API:
+
+    docker-compose --profile=pre-api-dev up
```
Comment (nutjob4life) [inline docker/README.md:41]: I think the Docker people are phasing out `docker-compose` in favor of the new integrated `compose` subcommand for `docker`, i.e., `docker compose`. We may wish to reflect that here.

**Review by nutjob4life — APPROVED** [review]
Seal of Approval

File: docker/README.md (hunk 1 above)
Comment (tloubrieu-jpl) [reply docker/README.md:41]: Thanks Sean, I will try that

Comment (tloubrieu-jpl) [reply docker/README.md:41]: docker compose works the same, I will update the README.

**Review by ramesh-maddegoda — APPROVED** [review]
Looks good!

**Pull request merged**